services/docker-compose.yml

```diff
@@ -936,6 +940,8 @@ services:
     environment:
       WEBSERVER_LOGLEVEL: ${WB_DB_EL_LOGLEVEL}
 
+      LONG_RUNNING_TASKS_NAMESPACE_SUFFIX: wb-db-event-listener
```

Comment:
the hostname defines these already. and are included in the APP_NAME. please use that and revert that.